dewey.properties.tmpl

```diff
@@ -3,6 +3,7 @@ dewey.environment-name          = {{ env "DE_ENV" }}
 
 {{ with $v := (key (printf "%s/irods-amqp/uri" $base)) }}dewey.amqp.uri = {{ $v }}{{ end }}
 {{ with $v := (key (printf "%s/irods-amqp/exchange/name" $base)) }}dewey.amqp.exchange.name = {{ $v }}{{ end }}
+{{ with $v := (key (printf "%s/dewey/amqp-queue-name" $base)) }}dewey.amqp.queue.name = {{ $v }}{{ end }}
```

Comment:
Oh! I found an issue, sorry. This says `queue.name` but everywhere else says `queue_name`. So those need to be the same.